.github/ct-install.yaml

```diff
@@ -1,6 +1,6 @@
 remote: origin
 target-branch: master
-helm-extra-args: --timeout 120s
+helm-extra-args: --timeout 300s
```

Comment:
```suggestion
helm-extra-args: --timeout 120s
```

Reply:
There are multiple pods and `wait` init contains, 120s is not always enough

Reply:
I'm okey with going for 180

5 minutes(!) is too long though.